apps/melding-form/src/app/(general)/bijlage/Attachments.tsx

```diff
@@ -189,6 +211,10 @@ export const Attachments = ({ files, formData, meldingId, token }: Props) => {
     }
   }, [systemError])
 
+  useEffect(() => {
+    if (actionSystemError) setSystemError(handleApiError(actionSystemError))
```

Comment:
We don't really use the contents of `actionSystemError`, we only set the generic `SystemErrorAlert` when it exists. I don't think we need this `useEffect`, we can just render `SystemErrorAlert` when `systemError` or `actionSystemError` is truthy. And we can add `actionSystemError` to the `useEffect` on lines 206-212.

Reply:
Good one